src/build/__tests__/build-test.js

```diff
@@ -9,15 +9,30 @@ import {
   GraphQLBoolean,
   GraphQLID,
 } from 'graphql/type';
+import { DirectiveLocation, GraphQLDirective } from 'graphql/type/directives';
+
 import { parse } from 'graphql/language';
 import { expectTypesEqual, expectSchemasEqual } from './comparators';
 import build, { buildTypes } from '../';
 
+const customDirective = new GraphQLDirective({
+  name: 'Dir1',
+  description:
+  'My custom uppercase',
+  locations: [
+    DirectiveLocation.FIELD,
+  ],
+});
 const querySource = `
  type Query {
    ok: Boolean!
  }
 `;
+const querySourceWithDirective = `
+ type Query {
+   ok: Boolean! @Dir1
+ }
```

Comment:
It was my intention to have 2 spaces of indentation for the template string as in `timestampSource` for example. Could you update that for `querySourceWithDirective` and `querySource`? 😄